runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/wrappers/SourceInputFormat.java

```diff
@@ -109,14 +105,12 @@ public float getAverageRecordWidth() {
   @Override
   @SuppressWarnings("unchecked")
   public SourceInputSplit<T>[] createInputSplits(int numSplits) throws IOException {
-    long desiredSizeBytes;
     try {
-      desiredSizeBytes = initialSource.getEstimatedSizeBytes(options) / numSplits;
-      List<? extends Source<T>> shards = initialSource.splitIntoBundles(desiredSizeBytes,
-          options);
+      long desiredSizeBytes = initialSource.getEstimatedSizeBytes(options) / numSplits;
+      List<? extends Source<T>> shards = initialSource.splitIntoBundles(desiredSizeBytes, options);
       List<SourceInputSplit<T>> splits = new ArrayList<>();
       int splitCount = 0;
-      for (Source<T> shard: shards) {
+      for (Source<T> shard : shards) {
         splits.add(new SourceInputSplit<>(shard, splitCount++));
       }
       return splits.toArray(new SourceInputSplit[splits.size()]);
```

Comment:
not especially important but looks like a minor efficiency gain from preallocating an array instead of the list. You already have the size at L110.